### PR TITLE
ros1/ros2 test all topics data

### PR DESCRIPTION
**Pull request by makaveli10**

### CODE REVIEW

File: carla_ros_bridge/test/ros_bridge_client_ros2_test.py [hunk 1]
```diff
@@ -381,7 +442,7 @@ def test_radar(self):
             if node is not None:
                 node.destroy_node()
             roscomp.shutdown()
-
+
```
Comment (zoq) [inline carla_ros_bridge/test/ros_bridge_client_ros2_test.py]:
```suggestion

```

Remove the extra spaces.

File: carla_ros_bridge/test/ros_bridge_client.py [hunk 2]
```diff
@@ -129,43 +132,82 @@ def test_camera_image(self):
...
+        """
+        rospy.init_node('test_node', anonymous=True)
+        msg = rospy.wait_for_message(
+            "/carla/ego_vehicle/semantic_segmentation_front/camera_info", CameraInfo, timeout=TIMEOUT)
+        self.assertEqual(msg.header.frame_id, "ego_vehicle/semantic_segmentation_front")
         self.assertEqual(msg.height, 70)
         self.assertEqual(msg.width, 400)
-
-    def test_dvs_camera_image(self):
+
```
Comment (zoq) [inline carla_ros_bridge/test/ros_bridge_client.py]:
```suggestion

```

Remove the extra spaces.

**Review by zoq — COMMENTED** [review]
Can you deliberately alter a test so that it fails, just like to make sure the GitHub Actions will report the job as failed.

**Comment by makaveli10** [comment]
> Can you deliberately alter a test so that it fails, just like to make sure the GitHub Actions will report the job as failed.
 
@zoq Yes, I could do that. Do you want me to push a commit?

**Comment by zoq** [comment]
> > Can you deliberately alter a test so that it fails, just like to make sure the GitHub Actions will report the job as failed.
> 
> @zoq Yes, I could do that. Do you want me to push a commit?

Yes, if you can do that it would be great.

**Review by zoq — COMMENTED** [review]
Looks like it fails as planned, let's revert the commit and get this merged.

**Review by zoq — APPROVED** [review]
Looks good to me.

**Pull request merged**